File: Acadia_Graph/python/distShape.py
# distShape.py
import numpy as np
import math

def distShape(x,ind,mu,sigma,s):
    if s == 1:
        exp_arg = (-np.square((x-mu)))/(2*sigma**2) #check size of mu
        y = np.exp(exp_arg) / (sigma*math.sqrt(2*math.pi))
    elif s == 2:
        y = (1+np.tanh(sigma*(x-mu)))/2
    return y

File: Matlab-Stochastic/distShape.m
function y = distShape(x,mu,sigma,s)
%y = 2-2*x*exp(-x^2)/sqr(pi)+ erf(x);  
if s == 1
  %x
  %mu
  y = exp(-(x-mu).^2/(2*sigma^2))/(sigma*sqrt(2*pi));
  %y
elseif s == 2  
  y = (1+tanh(sigma*(x-mu)))/2;
elseif s == 3  
 y = zeros(size(x));
 y(x>=mu-sigma/2 & x<=mu+sigma/2) = 1/sigma;
else 
  y = zeros(size(x));
end 
end

